django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
In Travis, this showed
```
python3 /home/travis/build/pyvideo/data/.tests/schemas.py -d /home/travis/build/pyvideo/data -s /home/travis/build/pyvideo/data/.schemas -v 0
Error JSON-decoding /home/travis/build/pyvideo/data/django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json
make: *** [test-schemas] Error 1
```
See https://travis-ci.org/pyvideo/data/builds/245720689